docs/plugins/common.mdx

```diff
@@ -0,0 +1,709 @@
+| `englishName`  | string  | English name of the language                          |
+| `name`         | string  | Language name                                         |
+| `postgresLang` | string  | Character set type                                    |
+
+`Locale` Props:
+
+| Name            | Type   | Description                  |
+| --------------- | ------ | ---------------------------- |
+| `localizedName` | string | Localized name of the locale |
+| `name`          | string | Locale name                  |
+| `value`         | string | Language code                |
+
+### `modal`
+
+Utility module to use together with the [`modal`](components#modal) component.
+
+Enums:
+
+:::info
+
+The following enums are a shortcut to the [`Flex`](components#flex) component enums.
+
+:::
+
+| Enum              | Description                                                                      |
+| ----------------- | -------------------------------------------------------------------------------- |
+| `modal.Align`     | Flexbox alignments <br />`START` \| `END` \| `CENTER` \| `STRETCH` \| `BASELINE` |
+| `modal.Direction` | Flexbox directions <br />`VERTICAL` \| `HORIZONTAL` \| `HORIZONTAL_REVERSE`      |
+| `modal.Justify`   | Flexbox justifies <br />`START` \| `END` \| `CENTER` \| `BETWEEN` \| `AROUND`    |
+| `modal.Wrap`      | Flexbox wraps <br />`NO_WRAP` \| `WRAP` \| `WRAP_REVERSE`                        |
+
+Functions:
+
+| Name         | Parameters                                                                                                             | Return Type              | Description                                                                  |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------- | ------------------------ | ---------------------------------------------------------------------------- |
+| `alert`      | <Required /> `props`: `AlertProps`                                                                                     | void                     | Brings up a modal useful for a quick response                                |
+| `closeModal` | <Required /> `modalKey`: string <br /> `contextKey`: string                                                            | void                     | Closes a specific modal by its key                                           |
+| `confirm`    | <Required /> `props`: `AlertProps`                                                                                     | Promise<boolean \| null> | Brings up an alert modal useful for asking the user for a quick confirmation |
+| `openModal`  | <Required /> `render`: (props: `ModalProps`) => ReactNode <br /> `options`: `ModalOptions` <br /> `contextKey`: string | string                   | Opens a specific modal and returns its key                                   |
+
+:::info
```

Comment:
Change this to a prop object for consistency and just update the description saying the boolean will be whether "confirm" was pressed or not